docs/README.md

```diff
@@ -21,50 +22,69 @@ Whenever possible, Omnia will leverage existing projects rather than reinvent th
+Omnia can be deployed on clusters that already have an RPM-based Linux OS running on them and are all connected to the Internet. Currently, all Omnia testing is done on [CentOS](https://centos.org). Please see [Example system designs](EXAMPLE_SYSTEM_DESIGNS.md) for instructions on the network setup.
 
-Once servers have functioning OS and networking, you can use Omnia to install and start Slurm and/or Kubernetes. Please see [Install Omnia using CLI](INSTALL_OMNIA.md) for detailed instructions.  
+Once servers have functioning OS and networking, you can use Omnia to install and start Slurm and/or Kubernetes. For detailed instructions, see [Install Omnia using CLI](INSTALL_OMNIA.md). 
 
 # System requirements  
-Ensure the supported version of all the software are installed as per the following table and other versions than those listed are not supported by Omnia. This is to ensure that there is no impact to the functionality of Omnia.
+The following table lists the software and operating system requirements on the management station, manager, and compute nodes. To avoid any impact on the proper functioning of Omnia, other versions than those listed are not supported.  
 
-Software and hardware requirements  |   Version
+Requirements  |   Version
 ----------------------------------  |   -------
-OS installed on the management node  |  CentOS 7.9 2009
-OS deployed by Omnia on bare-metal servers | CentOS 7.9 2009 Minimal Edition
+OS pre-installed on the management station  |  CentOS 8.3
+OS deployed by Omnia on bare-metal Dell EMC PowerEdge Servers | CentOS 7.9 2009 Minimal Edition
 Cobbler  |  2.8.5
-Ansible AWX  |  15.0.0
+Ansible AWX  |  19.1.0
 Slurm Workload Manager  |  20.11.2
-Kubernetes Controllers  |  1.16.7
+Kubernetes on the management station  |  1.21.0
+Kubernetes on the manager and compute nodes	|	1.16.7 or 1.19.3
 Kubeflow  |  1
 Prometheus  |  2.23.0
-Supported PowerEdge servers  |  R640, R740, R7525, C4140, DSS8440, and C6420
+
+## Hardware managed by Omnia
+The following table lists the supported devices managed by Omnia. Other devices than those listed in the following table will be discovered by Omnia, but features offered by Omnia will not be applicable.
+
+Device type	|	Supported models	
+-----------	|	-------	
+Dell EMC PowerEdge Servers	|	PowerEdge C4140, C6420, C6520, R240, R340, R440, R540, R640, R650, R740, R740xd, R740xd2, R750, R750xa, R840, R940, R940xa
+Dell EMC PowerVault Storage	|	PowerVault ME4084, ME4024, and ME4012 Storage Arrays
+Dell EMC Networking Switches	|	PowerSwitch S3048-ON and PowerSwitch S5232F-ON
+Mellanox InfiniBand Switches	|	NVIDIA MQM8700-HS2F Quantum HDR InfiniBand Switch 40 QSFP56
+
 
 ## Software managed by Omnia
```

Comment:
Let's change this to "Software deployed by Omnia"

Reply:
Section title is updated as per the suggestion.